README.rst

```diff
@@ -69,7 +69,7 @@ Somewhere in your server's request handler code:
            format=Format.HTTP_HEADERS,
            carrier=request.headers,
        )
-       span = tracer.start_span(
+       span = tracer.start_manual_span(
```

Comment:
can this work with `start_span` being the normal mode of operations and have `start_manual` being the fallback?